assets/test/view_test.js

```diff
@@ -264,6 +264,67 @@ describe("View + DOM", function() {
     expect(form.querySelector("button").dataset.phxDisabled).toBeTruthy()
     expect(form.querySelector("input").dataset.phxReadonly).toBeTruthy()
   })
+
+  test("empty diff undoes refs and pending attributes", () => {
```

Comment:
This test was already written. I just moved it back because it was in the wrong group.